### PR TITLE
[graphql] consolidate instigator has & get calls on to workspace context

### PR DESCRIPTION
**Pull request by alangenfeld**
refactor to use selectors and do point lookups against the workspace context, allowing different context implementations more flexibility 

## How I Tested These Changes

existing coverage

### CODE REVIEW

**Comment by alangenfeld** [comment]
* **#25445** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/25445?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @alangenfeld and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: python_modules/dagster-graphql/dagster_graphql/implementation/fetch_sensors.py [hunk 1]
```diff
@@ -190,28 +184,19 @@ def get_sensor_next_tick(
...
 
-    if not repository.has_sensor(sensor_state.name):
-        return None
-
-    sensor = repository.get_sensor(sensor_state.name)
-
-    if not sensor_state.is_running:
+    if not graphene_info.context.has_sensor(selector):
         return None
+    sensor = graphene_info.context.get_sensor(selector)
```
Comment (gibsondan) [inline python_modules/dagster-graphql/dagster_graphql/implementation/fetch_sensors.py:199]: this came up on the other PR but this has then get pattern is a little weird (esp if they both interact with a cache potentially), why not get and then handle it being None and make it more atomic?

**Review by gibsondan — APPROVED** [review]
non-blocking comment inline

File: python_modules/dagster-graphql/dagster_graphql/implementation/fetch_sensors.py (hunk 1 above)
Comment (alangenfeld) [reply python_modules/dagster-graphql/dagster_graphql/implementation/fetch_sensors.py:199]: good call, will change

**Pull request merged**